### PR TITLE
[#2850] Extend bundles functionality

### LINKED ISSUE #2850: New bundle design
Implement new bundle design according to the previews.
https://www.figma.com/file/0FiUXGZWNtRVgzu9ide5fh/Bundle?node-id=15%3A1165

Bundle Filed
**Name** - string [required]
**Bundle goals** - List of controlled values [required]: 
_Processing
Publishing
Anonymistaion
Data Storage
Computing Resources
Discover & Re-use Data
Data Management
Produce Data_
**Capabilities of goals** - List of controlled values [required]: 
_Data Transfer, 
Compute, 
Processing Tool, 
Storage, 
Big Data Storage, 
Long Term Storage, 
Anonimisation, 
Pseudonimisation, 
Metadata Removal Tool, 
Data Discoverability, 
Data Download, 
Data Sources, 
Publishing Service, 
DOI, 
Free, 
Domain Availability, 
Data Security, 
Long Term 
Publishing,
Other_ 
**Capabilities of goals suggestions** - string, filed visible only when user selects "Other" for **Capabilities of goals**, filed visible only in Backoffice
**Main offer** - Offer of the service for which the bundle is created [required]
**Description** - Text area [required]
**Order type** -  List of controlled values: full open access, open access, order required, other [required]
**Target users** -  List of controlled values from MP [required]
**Scientific domains** -  List of controlled values from MP [required]
**Research steps** -  List of controlled values from MP [required]
**Offers** - All offers in the system [except for the previously selected main offer [required]
**Related training url** - string [optional]
**Contact email** - validate e-mail address [optional]
**Helpdesk url** -  validate url address [required]

For bundle offer we don't display Order type on UI, change to "Bundle offer"
image

### PR DESCRIPTION
**Pull request by goreck888**
Add Bundle model with additional fields
Add form to create bundles
Change views to displaying data
from bundle model instead of
offers (eg. name, description, etc.)

Closes #2850

https://www.figma.com/file/0FiUXGZWNtRVgzu9ide5fh/Bundle?node-id=15%3A1165&t=iscNbxA0UQb5eabk-0

recent view: https://www.figma.com/file/0FiUXGZWNtRVgzu9ide5fh/Bundle?node-id=233-2896&t=63od3mPJoOR4IEpG-0

bundle exec rake rdt:add_internal_vocabularies

### CODE REVIEW

**Comment by Marcelinna** [comment]
when i click "edit" button on created bundle
image

**Comment by Marcelinna** [comment]
lack of Technical parameter
image

**Comment by Marcelinna** [comment]
when I don't fill all data and click on button "Create bundle"
image

**Comment by Marcelinna** [comment]
make filed mandator according with https://docs.google.com/spreadsheets/d/15pBonVmG0Vvz6uTq_ioJz0cpj3g5J3O0XeE6cnbqZQg/edit#gid=1974691285

**Comment by Marcelinna** [comment]
@agpul @eldenkat 
Please add a message which fields in bundle are multi-select values ​​and their dictionary values

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
I created bundle from 3 offers
image

When I am in order details I see only 2 offers
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

When I order bundle, its name should be visible in order step
image
image

same as we have for other offers
image

**Review by agpul — CHANGES_REQUESTED** [review]
image
 I can't fill this fields (no list) and it cause that I can't create the bundle

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

In bundle offer can't set order type -> order required internal

Simple offer
image

Bundle offer
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

When I create bundle for service and I want to click button "Access service"

image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
image
Change to You can add a dedicated bundle for
your **service**

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
Add notification near Main offer [Main offer must be a offer which uses EOSC Portal as the order management platform]
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

1. I create 2 additional offers for Service Aiidalab (service has 3 offer)
image
2. Click "Add new bundle"
3. I can't find 2 other offers for the Service Aiidalab
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
Beta
image
PR when I have open access type I see checkbox "Use EOSC Portal as the order management platform" 
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

offer created for bundle looks weird, should look like a normal offer
image

File: app/views/layouts/common_parts/services/_new_bundle.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      %p
+        = image_pack_tag "icon-tag.png", class: "icon-tag"
+        You can add a dedicated bundle for
+        %br your service
+
+    .card-button.text-center
+      %label
+        = link_to new_bundle_link(service, controller_name), "data-e2e": "add-new-offer-btn" do
+          %span.btn.btn-outline-secondary.font-weight-bold
+            = _("Add new bundle")
```
Comment (Marcelinna) [inline app/views/layouts/common_parts/services/_new_bundle.html.haml:13]: please change to "data-e2e": "add-new-bundle-offer-btn"

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

Bundle for EGI Nootebook is removed
When I click Access to service on EGI Nottebook page
image

**Review by Marcelinna — APPROVED** [review]

**Review by agpul — APPROVED** [review]

**Pull request merged**